src/sentry/search/events/fields.py

```diff
@@ -2165,7 +2165,7 @@ class MetricArg(FunctionArg):
     def __init__(
         self,
         name: str,
-        allowed_columns: Sequence[str] | None = None,
+        allowed_columns: Sequence[str] | set[str] | None = None,
```

Comment:
I believe this should be something like `Iterable[str]` or `Container[str]` instead of `Sequence | set` (which is usually a mistake)